README.md

```diff
@@ -356,6 +356,31 @@ status:
 ```
 > NOTE: The supported enforcementActions are [`deny`, `dryrun`] for constraints. Update the `--disable-enforcementaction-validation=true` flag if the desire is to disable enforcementAction validation against the list of supported enforcementActions.
 
+### Exempting Namespaces from Gatekeeper
```

Comment:
I think we should clarify that this is just for admission, not auditing. For auditing, users would need to add `namespace` or `namespaceSelector` to each constraint in order to include or exclude namespaces.

Reply:
+1, changed.